### ISSUE TITLE
why to use Resent-Sender header? 

### ISSUE THREAD

**Issue by skob** [atech/postal#767]
Some mail providers (Mail.Ru for example) show info from Resent-Sender header into main interface.
Maybe Return-Path is enough?

**Comment by willpower232** [comment]
Postal shows the message headers for each message so if the message includes a Resent-Sender header then it should be visible there or are you asking something else?

**Comment by skob** [comment]
<img width="351" alt="Снимок экрана 2019-03-14 в 20 44 07" src="https://user-images.githubusercontent.com/4798932/54379329-3f27ed80-469a-11e9-98f6-0b6ba67e2eec.png">

something like this ...

**Comment by skob** [comment]
And `psrp.<domain>` is valid , of course

**Comment by willpower232** [comment]
If Postal receives a bounce message at the psrp address, this is currently shown in the original messages UI like this:

![image](https://user-images.githubusercontent.com/1619102/54422547-a8dde100-4706-11e9-9d4a-6f548bb20534.png)

The bounce message is linked to and the headers are in the tab as I said before.

Apart from webhooks, I don't think Postal does anything else with bounced emails.

Is that what you're after?

**Comment by skob** [comment]
mmm
I think it's problem with my explanation

Let me try again:

It is header: 
https://github.com/atech/postal/blob/master/lib/postal/smtp_sender.rb#L123
really needed?
it's break UI for some mail providers :-)

**Comment by willpower232** [comment]
Ah sorry I see what you mean now. Is that the mail.ru web interface or a mail client like outlook?

**Comment by skob** [comment]
it is web interface

but I'm affraid some mail clients can try to parse this header too

**Comment by catphish** [comment]
This header is present to appease certain recipients who would otherwise be unhappy with the envelope sender not matching the from address. Unfortunately the fact that some providers list the envelope sender in their UI is rather unavoidable. In this specific case, it might be that removing the header would solve the mail.ru problem, but I would prefer not to manage rulesets for known recipient providers if at all possible.

**Comment by skob** [comment]
Hm
So which options I have?..

**Comment by catphish** [comment]
You could just ignore the fact that the header is displayed, or you could delete the line of code that adds it.

**Comment by catphish** [comment]
Actually it's not quite as simple as deleting a line. The code that adds it is here:

https://github.com/atech/postal/blob/ed7889a40e1d777f1d80821043562ab18b04766e/lib/postal/smtp_sender.rb#L123

The line could be changed to

```
raw_message = message.raw_message
```

And then the resent-header would not be inserted.

**Comment by skob** [comment]
Thank you , I know — see start question :-)
but local change potentially break updates and so on
